data/hai/unfettered.txt

```diff
@@ -0,0 +1,161 @@
+	"<, unfettered title>" ", Herald"
+		has "salary: Unfettered"
+	"<, unfettered title>" ", Warlord"
+		"salary: Unfettered" > 1000
+	"<, unfettered title>" ", High Warlord"
+		has "unfettered: high warlord"
+	"<, unfettered title>" ", outcast"
+		has "unfettered: outcast"
+
+
+
+conversation "unfettered outcast"
+	branch "no license"
+		has "license: Unfettered"
+	action
+		# replace it by a wortless outfit
+		clear "license: Unfettered"
+		outfit "Unfettered License"
+	label "no license"
+	action
+		event "unfettered: outcast"
+		event "unfettered: challengers leave"
+	`The local authorities hail you immediately after the destruction of the Alpha ship with a code purple.`
+		to display
+			has "unfettered: destroyed auxiliary"
+	`The local authorities hail you immediately after the destruction of the duelling ship with a code purple.`
+		to display
+			not "unfettered: destroyed auxiliary"
+	branch translator
+		has "language: Hai"
+	`	They speak in Hai but the tone alone is enough to understand what they mean: you have angered them greatly by destroying that ship, and you should leave, or you risk being shot at. You probably will not be offered missions from them in the future.`
+		decline
+	label translator
+	branch "alpha ship"
+		has "unfettered: destroyed auxiliary"
+	`	"You have disappointed us greatly by breaking the sacred rules of the duels! Think very carefully before you make any more mistakes, or it will be war! We have removed any privileges you had, and have added you to the list of shame. Rest assured that no True Hai will offer you special missions any longer, but we'll allow you to ferry cargo. For now."`
+		decline
+	label "alpha ship"
+	`	"How dare you?! This ship was helping us by providing supplies, and you just shoot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first shot, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`
+
+
+
+event "unfettered: outcast"
+	set "unfettered: outcast"
+	planet "Firelode"
+		remove outfitter "Mera's Lab"
+	planet "Darkcloak"
+		remove outfitter "Unfettered"
+	government "Hai (Unfettered)"
+		"bribe" .04
+	"reputation: Hai (Unfettered)" <?= 0
+
+
+
+mission "Unfettered: Regrets"
+	landing
+	invisible
+	to offer
+		not "salary: Unfettered"
+		has "Wanderers: Defend Vara Ke'sok: done"
+		not "unfettered: outcast"
+	on enter "Ehma Ti" "Wah Yoot" "Hi Yahr"
+		event "unfettered: outcast"
+		dialog
+			`The instant you enter the system, you receive a priority message from the local authorities. "<last>, because of your recent involvements with the aliens up north, we have decided to stop offering you important missions."`
+			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you. You assume that your Unfettered license no longer has any value to them.`
```

Comment:
```suggestion
			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you. You assume that your Unfettered license no longer has any value to it.`
```

Reply:
I meant to the uhai

Isnt it better to just remove the two last words tho?